homeassistant/components/bluetooth/__init__.py

```diff
@@ -315,23 +315,21 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     mode = BluetoothScanningMode.PASSIVE if passive else BluetoothScanningMode.ACTIVE
     manager: HomeAssistantBluetoothManager = hass.data[DATA_MANAGER]
     scanner = HaScanner(mode, adapter, address)
+    scanner.async_setup()
```

Comment:
setup cannot raise anymore, bleak scanner is created in async_start which is where the RuntimeError can come from